### ISSUE TITLE
MySQL/PostgreSQL: Backup All Databases	

### ISSUE THREAD

**Issue by rostovtsev** [webmin/webmin#970]
Jamie, this doesn't work correctly in neither themes.

It returns an error anytime you click _Backup Now_ with _Download to browser_ option.

Later, it returns an error "Database backup failed : Only a single database backup can be downloaded".

In old themes it just dies. In Authentic Theme, if you click backup, it shows and error, and if you hit _go back_ - it takes you back _**BUT**_ this time it shows the select with databases and let's you select single database.

I think it's better to backup each database individually, then archive it and push for user to download and make all databases downloadable in a single go.

**Comment by jcameron** [comment]
Yeah, that's a missing feature really - if you backup multiple databases to the browser and download them, it should produce a ZIP file.

**Comment by iliaross** [comment]
❗️ If the issue isn't resolved, feel free to reopen it with updated details.
